src/services/carNamesTrimmer.js

```diff
@@ -0,0 +1,22 @@
+const splitCarNames = (carNames) => {
+  const eachCarName = carNames.split(',');
+  return eachCarName;
+};
+
+const trimEachCarNames = (eachCarName) => {
+  const trimmedEachCarName = [];
+
+  for (let i = 0; eachCarName.length > i; i += 1) {
+    trimmedEachCarName[i] = eachCarName[i].trim();
+  }
+
+  return trimmedEachCarName;
```

Comment:
자바스크립트의 내장함수를 쓰면 for문을 쓰지 않고도 좀 더 간결하게 코드를 작성할 수 있습니다!
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/map
유용한 함수들이 많으니 구현하시기 전에 찾아보시는 것도 좋을 것 같아요~

```suggestion
  const trimmedEachCarName = eachCarName.map((name) => name.trim())
  return trimmedEachCarName;
```

Reply:
감사합니다! 이 부분은 다른 분도 피드백 하시더라고요! 프리코스 3에서 적용 꼭 해보겠습니다!